library/init/meta/interactive.lean

```diff
@@ -133,813 +132,3 @@ meta def param_desc : expr → tactic format
   then return $ to_fmt "{ tactic }"
   else paren <$> pp e
 end interactive
-
```

Comment:
Is everyone fine with this refactoring to reduce dependencies? If yes, I'd recommend merging the PR as-is to reduce merge conflicts. I will improve error handling in a later PR.